src/main/resources/avro/PingPongBallEvent.avsc

```diff
@@ -0,0 +1,11 @@
+{
+  "namespace": "com.cedrus.patpetillo.springkafkapingpong.avro",
+  "type": "record",
+  "name": "PingPongBallEvent",
+  "doc": "Represents a Ping Pong Ball Event",
+  "fields": [
+    {"name":  "id", "type": "string"},
+    {"name":  "pingPongTeam", "type": "string"},
```

Comment:
pingPongTeam isn't very specific.  Is this the team that caused the event?  The team towards which the ball is hit?  Maybe your object should have both.